.github/workflows/release.yml

```diff
@@ -13,6 +13,11 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
 
+      - name: Trigger Manpage Generation
+        uses: ./.github/workflows
+        with:
+          workflow: generate_manpages.yml
```

Comment:
Nitpick: it's just one manpage we generate. You can get rid of the plural.

Reply:
I have changed the name to be singular!